--- highway-0.16.0/hwy/nanobenchmark.cc.orig	2022-04-09 18:30:28.550190500 +0200
+++ highway-0.16.0/hwy/nanobenchmark.cc	2022-04-09 18:31:11.596623964 +0200
@@ -439,6 +439,12 @@ HWY_DLLEXPORT double Now() {
 }
 
 HWY_DLLEXPORT uint64_t TimerResolution() {
+#if HWY_ARCH_X86
+  /* TimerResolution() is used to initialize static variable, used in Measure() function */
+  if (!platform::HasRDTSCP())
+    return 0;
+#endif
+
   // Nested loop avoids exceeding stack/L1 capacity.
   timer::Ticks repetitions[Params::kTimerSamples];
   for (size_t rep = 0; rep < Params::kTimerSamples; ++rep) {
